joomla/joomla-cms
# [4.0] Remove adminlist.css

Pull Request for Issue # .

### Summary of Changes

This PR removes an old CSS file. As far as I can tell, it's not used or needed.

### Testing Instructions

1. Login to the frontend of your Joomla site.
2. Go to `/index.php/submit-article`
3. In the TinyMCE editor toolbar, expand the `CMS Content` dropdown item and click `Menu`

This will display a modal.

Ensure the styling of the content
